drivers/at/at.c

```diff
@@ -174,7 +174,7 @@ int at_send_cmd(at_dev_t *dev, const char *command, uint32_t timeout)
     uart_write(dev->uart, (const uint8_t *)command, cmdlen);
     uart_write(dev->uart, (const uint8_t *)CONFIG_AT_SEND_EOL, AT_SEND_EOL_LEN);
 
-    if (AT_SEND_ECHO) {
+    if (!IS_ACTIVE(CONFIG_AT_SEND_SKIP_ECHO)) {
```

Comment:
Since `CONFIG_AT_SEND_SKIP_ECHO` is always set, you can just

```suggestion
    if (!CONFIG_AT_SEND_SKIP_ECHO) {
```

which is easier to read

Reply:
I think using `IS_ACTIVE` is ok here (and widely used elsewhere)

Reply:
Well, I've been thinking of this and keeping the define in at.h only for Doxygen is ok and justify the use of `IS_ACTIVE` here. I think this is why it was done like this in the deprecation PR.